apps/src/templates/lessonOverview/StudentLessonOverview.jsx

```diff
@@ -114,6 +130,10 @@ class StudentLessonOverview extends Component {
             />
           </div>
         )}
+        <div id="level-section">
+          <h2>{i18n.levels()}</h2>
+          {this.determineLevelDisplay()}
```

Comment:
Nit: on LP we've decided that functions in components that render something should be prefixed with `render-`, a potential name for this function could be `renderLevels`. Alternatively there isn't much logic in the function so you could include that logic here directly in the render.

Reply:
Gotcha- this makes sense. I've updated to renderLevels().